CONTRIBUTING.md

```diff
@@ -230,12 +231,14 @@ Each source file needs to have this header:
     // See the License for the specific language governing permissions and
     // limitations under the License.
 ```
-Note: `DATE` is either a year or a range of years with the first and last years of the range separated by a comma. So for example: "2004" or "2000, 2004". The first year is when the contents of the file were first created and the last year is when the contents were last modified.
+Note: `DATE` is either a year or a range of years with the first and last years of the range separated by a dash. So for example: "2004" (initial and last contribution in the same year) or "2000 - 2004". The first year is when the contents of the file were first created and the last year is when the contents were last modified. The year of the last contribution needs to be updated the content was modified in a new year. An option is to add a year by a comma for the case that a contributor did not modify in one year.
```

Comment:
> The year of the last contribution needs to be updated the content was modified in a new year.

This is confusing. How about "The last modified date must be updated according to the current year of the contribution."

Same with the following sentence. How about "If there is a contribution gap of one or more calendar years, the current year can be appended with a comma to separate it from the last contribution."

Maybe we need to ask a native speaker like @ithier or almost native like @marthtz

Reply:
Yes, this could be improved. Besides this LGTM

Reply:
How about:
"The last mentioned date needs to be the last year of the contribution. If there is a contribution gap of one or more calendar years in the range, the current year can be appended with a comma to separate it from the last contribution. (E.g. "2000 - 2004, 2006")"

Reply:
since it relates to the previous sentence it could also be something like "Which means the latter needs to be updated with your contribution."

Reply:
"The years of contribution should be ordered in chronological order, thus the last date in the list should be the year of the most recent contribution. If there is a gap between contributions of one or more calendar years, use a comma to separate the disconnected contribution periods. (e.g. "2000 - 2004, 2006")"

Take or leave whatever you think fits best.

Reply:
@ithier sounds great!